setup.pl

```diff
@@ -121,8 +126,17 @@
 	open(VAR, "$config_directory/var-path");
 	chop($var_dir = <VAR>);
 	$var_directory = $var_dir;
+	$ENV{'WEBMIN_VAR'} = $var_directory;
 	close(VAR);
 
+	# Get current bootscript name
+	if (-r "$config_directory/bootscript-name") {
+		open(BOOTVAR, "$config_directory/bootscript-name");
+		chop($bootscript = <BOOTVAR>);
+		close(BOOTVAR);
+		$bootscript ||= ($ENV{'bootscript'} || "webmin");
```

Comment:
Shouldn't this be moved outside the if statement, so $bootscript is always set?

Reply:
No, as this is a global variable. It will be stored either way.

Reply:
But is $bootscript set to some default higher up if `bootscript-name` doesn't exist?

Reply:
Yes, it's the same as in `setup.sh`, and already set on the line 12 of `setup.pl` ..

Reply:
Ok cool